strawberry/extensions/query_depth_limiter.py

```diff
@@ -129,12 +129,13 @@ def __init__(
                 )
             validator = create_validator(max_depth, should_ignore, callback)
         else:
-            warnings.warn(
-                "The `ignore` argument to `QueryDepthLimiter` is deprecated. "
-                "Please use `should_ignore` instead.",
-                DeprecationWarning,
-                stacklevel=1,
-            )
+            if ignore is not None:
```

Comment:
should we do elif?

also I'm not opposed to removing this argument entirely, since we've deprecated for a while now 😊

Reply:
Hey patrick! 👋 

Sure! Sounds good to me! I will modify the PR to remove the arg 👍 Thanks!

Reply:
Done! Thank you!